docs/maui/views/MediaElement.md

```diff
@@ -173,6 +173,20 @@ remote URL with artwork for the lockscreen. It should be at least 1080P for best
 > You can set the metadata in either XAML or code behind. If you are setting it in code behind you need to set the source in code behind. The source should
 be set last. If you set the metadata in XAML or in constructor this note can be safely ignored.
 
+### Using TextureView
+
+A `MediaElement` can use `TextureView` on Android Platform. The platform default behavior is to use `SurfaceView`. Using Texture View adds support
+to allow transparencies and other effects. This is set by changing the builder to use
+
+```csharp
+.UseMauiCommunityToolkitMediaElement(options => {
+				options.AndroidViewType = AndroidViewType.TextureView;
+})
```

Comment:
Please update this to use the updated API name and to use [static anonymous functions](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/static-anonymous-functions)

```suggestion
.UseMauiCommunityToolkitMediaElement(static options => 
{
				options.SetDefaultAndroidViewType(AndroidViewType.TextureView);
})
```